File: c/estrutura_condicional/notas.c
/* Problema "notas"
 Fazer um programa para ler as duas notas que um aluno obteve no primeiro e segundo semestres de
 uma disciplina anual. Em seguida, mostrar a nota final que o aluno obteve (com uma casa decimal) no
 ano juntamente com um texto explicativo. Caso a nota final do aluno seja inferior a 60.00, mostrar a
 mensagem "REPROVADO", conforme exemplos.
*/

#include <stdio.h>

int main()
{
  double a, b, finalGrade;

  printf("Digite a primeira nota: ");
  scanf("%lf", &a);

  printf("Digite a segunda nota: ");
  scanf("%lf", &b);

  finalGrade = a + b;
  
  printf("NOTAL FINAL = %.1lf\n", finalGrade);

  if (finalGrade >= 60)
  {
    printf("APROVADO!\n");
  }
  else
  {
    printf("REPROVADO!\n");
  }

  return 0;
}

File: csharp/Source/Estruturas/Condicional/Notas.cs
namespace Source.Estruturas.Condicional;

/**
 * Problema "notas"
 * Fazer um programa para ler as duas notas que um aluno obteve
 * no primeiro e segundo semestres de uma disciplina anual.
 * Em seguida, mostrar a nota const que o aluno obteve (com uma casa decimal)
 * no ano juntamente com um texto explicativo.
 * Caso a nota const do aluno seja inferior a 60,00,
 * mostrar a mensagem "REPROVADO", conforme exemplos.
 */

internal static class Notas
{
    private static readonly double GRADE_CRITERIA = 60.0;

    public static void Execute()
    {

        Console.Write("Digite a primeira nota: ");
        double gradeA = double.Parse(Console.ReadLine()!);

        Console.Write("Digite a segunda nota: ");
        double gradeB = double.Parse(Console.ReadLine()!);

        double constGrade = gradeA + gradeB;

        Console.WriteLine("NOTAL const = {0:F1}", constGrade);

        if (constGrade >= GRADE_CRITERIA)
        {
            Console.WriteLine("APROVADO!");
        }
        else
        {
            Console.WriteLine("REPROVADO!");
        }
    }

}